jax/experimental/ode.py

```diff
@@ -448,7 +449,7 @@ def onearg_odeint(args):
   dim = 10
   t0 = 0.1
   t1 = 0.2
-  y0 = np.linspace(0.1, 0.9, dim)
+  y0 = np.linspace(0.1, 0.9, dim).reshape((5, 1, 2))
```

Comment:
Instead of changing existing tests, would it be possible to add new tests?

Reply:
Done! 95d20d24109d86f66f4465014fa040c422fcf05d